gravityflow/gravityflowwoocommerce
# Fix the "Create Entry on Specific Statuses" setting is not working

re: [HS#7195](https://secure.helpscout.net/conversation/711427960/7195?folderId=2308453)

This PR fixes an issue that the "Create Entry on Specific Statuses" setting isn't working since PR #10.

## Testing instructions
1. Logged in to the dev site http://dev-wcgflow.pantheonsite.io/ and checkout with a failed card (ends with 0341).
2. No new entries should be created in http://dev-wcgflow.pantheonsite.io/wp-admin/admin.php?page=gf_entries&id=4
3. Checkout with another card (ends with 4242), new entries should be created now.
